### PR TITLE
Fix vocalisation of child removal button

### LINKED ISSUE #292: Vocaliser correctement le bouton d'effacement d'un enfant
En tant que déficient visuel,
Je peux comprendre l'action associée au bouton de suppression d'un enfant,
Pour ne pas l'actionner par erreur

- - - - -

Actuellement vocalisé "Bouton fois".

### PR DESCRIPTION
**Pull request by MattiSG**
Fix #292.

Tested with VoiceOver.

### CODE REVIEW

File: app/views/partials/foyer/enfants.html [hunk 1]
```diff
@@ -13,7 +13,8 @@
           <div class="panel-body">
             {{ enfant.firstName }}
             <button type="button" class="close" ng-click="removeEnfant(enfant)">
-              <span aria-hidden="true">&times;</span>
+              <span class="screenReadersOnly">Effacer</span>
```
Comment (fpagnoux) [inline app/views/partials/foyer/enfants.html]: Nice class, right 😎  ?

File: app/views/partials/foyer/enfants.html [hunk 2]
```diff
@@ -13,7 +13,8 @@
...
+              <span aria-role="presentation" aria-hidden="true">&times;</span>
```
Comment (fpagnoux) [inline app/views/partials/foyer/enfants.html]: > So is-it a good practise to add `aria-role="presentation"` when using `aria-hidden="true"`?

Comment (MattiSG) [inline app/views/partials/foyer/enfants.html]: To be honest, I don't know exactly. I used to think `role="presentation"` was enough, but [it seems not](http://john.foliot.ca/aria-hidden/).

Comment (MattiSG) [inline app/views/partials/foyer/enfants.html]: This should be `role`, not `aria-role` :-/

File: app/views/partials/foyer/enfants.html (hunk 1 above)
Comment (MattiSG) [inline app/views/partials/foyer/enfants.html]: We could improve vocalisation by adding the name of the child.

**Comment by fpagnoux** [comment]
Tested after vocalisation improve, GTM.

**Pull request merged**